.github/workflows/focus-ios-smoke

```diff
@@ -0,0 +1,124 @@
+name: "Focus/Firefox Tests"
+    
+on:
+    workflow_dispatch:
+
+env:
+    browser: focus-ios
+    xcode_version: 15.4
+    ios_version: 17.4
+    ios_simulator: iPhone 15
+    xcodebuild_test_plan: SmokeTest
+    xcodebuild_scheme: Focus
+    xcodebuild_target: XCUITest
+    test_results_directory: /Users/runner/tmp
+    
+jobs:
+    run-tests:
+        name: Build and run tests
+        runs-on: macos-14
+        steps:
+            - name: Check out source code
+              uses: actions/checkout@v4.1.7
+            - name: Setup Python
+              uses: actions/setup-python@v5.1.0
+              with:
+                python-version: '3.11'
+            - name: Install packages
+              id: packages
+              run: |
+                brew update
+                pip3 install virtualenv pipenv
+                brew install xctesthtmlreport
+            - name: Setup Xcode
+              id: xcode
+              run: |
+                sudo rm -rf /Applications/Xcode.app
+                sudo xcode-select -s /Applications/Xcode_${{ env.xcode_version }}.app/Contents/Developer
+                xcodebuild -version
+                xcrun simctl shutdown all
+                xcrun simctl erase all
+                ./checkout.sh
+                ./bootstrap.sh --force
+            - name: Compile source code
+              id: compile
+              run: |
+                xcodebuild \
+                    -resolvePackageDependencies \
+                    -onlyUsePackageVersionsFromResolvedFile
+                xcodebuild \
+                    build-for-testing \
+                    -scheme ${{ env.xcodebuild_scheme }} \
+                    -target ${{ env.xcodebuild_target }} \
+                    -destination 'platform=iOS Simulator,name=${{ env.ios_simulator }},OS=${{ env.ios_version }}'
+              working-directory: ${{ env.browser }}
+            - name: Run tests
+              id: run-tests
+              run: |
+                xcodebuild \
```

Comment:
No tests are run in parallel at this point. I just want to see if the tests are stable in Github Action's runner.